src/components/custom-field-input-date/custom-field-input-date.test.jsx

```diff
@@ -1,91 +1,107 @@
 import React, { createRef } from 'react';
 import {
-  cleanup,
   fireEvent,
   render,
   screen,
 } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
-import renderer from 'react-test-renderer';
 import CustomFieldInputDate from './custom-field-input-date.jsx';
 
 describe('src/components/custom-field-input-date/custom-field-input-date', () => {
-  const renderComponent = (props = {}) => render(<CustomFieldInputDate label="Field Date" id="field-date" {...props} />);
+  const requiredProps = {
+    id: 'field-date',
+    label: 'Field Date',
+    name: 'field-id',
+  };
+
   const changeValue = (getInputElement, value) => {
     userEvent.click(getInputElement());
     fireEvent.change(getInputElement(), { target: { value } });
     fireEvent.blur(getInputElement());
   };
 
-  afterEach(cleanup);
-
   it('has defaults', () => {
-    const tree = renderer.create(<CustomFieldInputDate label="Hello" id="hello" />).toJSON();
-    expect(tree).toMatchSnapshot();
+    const ref = createRef();
+    render(<CustomFieldInputDate {...requiredProps} ref={ref} />);
+    expect(document.body).toMatchSnapshot();
+    expect(ref.current).toMatchSnapshot();
+  });
+
+  describe('dirty ref API', () => {
+    xit('updates on user interactions', () => {
```

Comment:
Is this still supposed to be skipped?

Reply:
I haven't found a good way to test this. We do have a follow up story to integrate the component into our custom form. I think that will be a good time to deal with this -- nice and focused.

Reply:
/cc @bennapp .